github-user-card/src/components/FollowerCard.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+import CardMedia from '@material-ui/core/CardMedia';
+import CardContent from '@material-ui/core/CardContent';
+
+function FollowerCard(props){
+
+    return(
+      <div className="followers">
+          <h2>{props.data.name}'s Followers: </h2>
+        {props.followers.map(follower => {
```

Comment:
Fun fact, you can destruct in an `arr.map()` too! for this instance it would look something like this```props.followers.map(({ avatar_url, name, login, url }) => {})``` and you wouldn't have to reference everything in follower with a `follower.` before it! keep it DRY 🐪